src/script/ceph-debug-docker.sh

```diff
@@ -134,6 +134,7 @@ EOF
                 base_debuginfo="glibc-debuginfo"
                 ceph_debuginfo="ceph-base-debuginfo"
                 debuginfo=/etc/yum.repos.d/CentOS-Linux-Debuginfo.repo
+                base_url="s|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g"
```

Comment:
currently that line is commented out ('#')?

Reply:
Yes, you can cat `/etc/yum.repos.d/CentOS-Linux-Debuginfo.repo` for similar output.